python/tests/helpers/test_io.py

```diff
@@ -0,0 +1,24 @@
+"""
+Unit tests for the aac.util module.
+"""
+
+from os import getcwd
+from unittest import TestCase
+from tempfile import TemporaryDirectory
+from tests.helpers.io import new_working_dir
+
+
+class TestArchUtil(TestCase):
+    """
+    Unit test class for aac.util module.
+    """
+
+    def test_new_working_dir(self):
+        """
+        Unit test for the util.new_working_dir method.
+        """
+        original_working_directory = getcwd()
+
+        with TemporaryDirectory() as temp_directory, new_working_dir(temp_directory):
+            self.assertNotEqual(original_working_directory, temp_directory)
+            self.assertEqual(getcwd(), temp_directory)
```

Comment:
We should also check that outside of the context manager, the current working directory is back to `original_working_directory`.

```suggestion
            self.assertEqual(getcwd(), temp_directory)

        self.assertEqual(getcwd(), original_working_directory)
```

Reply:
Good call. :)